src/Plugin/Block/FooterBlock.php

```diff
@@ -81,9 +94,59 @@ public function build() {
 
     NestedArray::setValue($build, ['#corporate_footer', 'bottom_links'], $config->get('bottom_links'));
 
+    $general_links = $this->getCustomFooterLinks('footer_link_general', $cache);
+    $social_links = $this->getCustomFooterLinks('footer_link_social', $cache);
+    $site_info_config = $this->configFactory->get('system.site');
+    $cache->addCacheableDependency($site_info_config);
+    $site_identity = $site_info_config->get('name');
+
+    if (!empty($social_links) || !empty($general_links)) {
+      NestedArray::setValue($build, ['#custom_footer', 'site_identity'], $site_identity);
+      NestedArray::setValue($build, ['#custom_footer', 'social_links'], $social_links);
+      NestedArray::setValue($build, ['#custom_footer', 'other_links'], $general_links);
```

Comment:
Same as above, let's not use `custom` to refer to this functionality, let's use `#site_specific_footer`.